src/CommerceDashboardItemManager.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Drupal\acm;
+
+use Drupal\Core\Cache\CacheBackendInterface;
+use Drupal\Core\Extension\ModuleHandlerInterface;
+use Drupal\Core\Plugin\DefaultPluginManager;
+
+/**
+ * Provides the CommerceRequirementsManager plugin manager.
+ *
+ * @package acm
+ */
+class CommerceDashboardItemManager extends DefaultPluginManager {
+
+  /**
+   * CommerceRequirementManager constructor.
+   *
+   * @param \Traversable $namespaces
+   *   An object that implements \Traversable which contains the root paths
+   *   keyed by the corresponding namespace to look for plugin implementations.
+   * @param \Drupal\Core\Cache\CacheBackendInterface $cache_backend
+   *   Cache backend instance to use.
+   * @param \Drupal\Core\Extension\ModuleHandlerInterface $module_handler
+   *   The module handler.
+   */
+  public function __construct(\Traversable $namespaces, CacheBackendInterface $cache_backend, ModuleHandlerInterface $module_handler) {
+    parent::__construct(
+      'Plugin/CommerceDashboardItem',
+      $namespaces,
+      $module_handler,
+      'Drupal\acm\CommerceDashboardItemInterface',
+      'Drupal\acm\Annotation\CommerceDashboardItem'
+    );
+
+    $this->alterInfo('acm');
```

Comment:
I think this should be something different otherwise if someone needs to implement it, this will look like **hook_acm_alter()** which looks very very generic.

Reply:
yeah, it's just copied code from acm_diagnostics for now